### PR TITLE
Fix typo + heading structure in README

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**
fixes typo + better sorting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-[![JS.ORG](https://img.shields.io/badge/js.org-+-FFE70B.svg?style=flat-square)](http://js.org)
+[![js.org](https://img.shields.io/badge/js.org-+-FFE70B.svg?style=flat-square)](http://js.org)
 [![Donate](https://img.shields.io/badge/Donate-for_registrar_fees-1F87FF.svg?style=flat-square&logo=open-collective&logoColor=fff)](https://opencollective.com/js-org)
 
 ## Cleanup Scripts
-
```
Comment (MattIPv4) [inline README.md:5]: Please revert these line break removals. There should always be an extra linebreak between headings and the content.

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -16,21 +16,20 @@ There are four options within the CLI:
...
 https://github.com/js-org-cleanup/simulated-js.org/issues/46
 
-### Example Generated Contact Issue
-
+#### Generated Contact Issue
 https://github.com/js-org-cleanup/simulated-automatic-contact/issues/12
 
-### Example Generated Pull Request
-
+#### Generated Pull Request
```
Comment (MattIPv4) [inline README.md:34]: You are still missing linebreaks between headings and content here

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (WilliamDavidHarrison) [reply README.md:34]: Sorry, my bad

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**